tools/http3-test/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "http3test"
+version = "0.1.0"
+authors = ["Lucas Pardue <lucaspardue.24.7@gmail.com>"]
+edition = "2018"
+
```

```suggestion
publish = false

```
so we don't accidentally upload this to crates.io (I'll need to do this for the other tool as well).